setup.cfg

```diff
@@ -0,0 +1,17 @@
+[metadata]
+name = python_stub
+author = Daniel Huppmann
+author_email = huppmann@iiasa.ac.at
+description = Template for creating python packages and Sphinx-based docs pages
+url = https://www.iiasa.ac.at
+
+[options]
+packages = python_stub
+install_requires =
```

Comment:
Per the setuptools-scm docs: https://github.com/pypa/setuptools_scm/#setupcfg-usage
```
setup-requires = 
    setuptools_scm
```

Reply:
I removed that per https://setuptools.readthedocs.io/en/latest/references/keywords.html

> Using setup_requires is discouraged in favor of PEP-518

Reply:
OK, then should instead add the pyproject.toml file specified by PEP 518: https://github.com/pypa/setuptools_scm/#pyprojecttoml-usage

Reply:
ok, keep that for another day... #2